pkgs/by-name/op/openrct2/package.nix

```diff
@@ -118,12 +110,17 @@ stdenv.mkDerivation {
   ];
 
   postUnpack = ''
-    mkdir -p $sourceRoot/data/assetpack
+    mkdir -p $sourceRoot/data/{object,sequence}
+    unzip -o ${objects} -d $sourceRoot/data/object
+    unzip -o ${openmsx} -d $sourceRoot/data
+    unzip -o ${opensfx} -d $sourceRoot/data
+    unzip -o ${title-sequences} -d $sourceRoot/data/sequence
```

Comment:
Could we use fetchzip and symlinks?

Reply:
I tried doing this but found it's easier to download & extract to the sourceRoot. There are duplicate directories in the archives.

Reply:
we could use some tools like lndir for that but likely not worth it.